src/Repository/RedirectRepository.php

```diff
@@ -7,14 +7,10 @@
 use Setono\SyliusRedirectPlugin\Model\RedirectInterface;
 use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
 
-class RedirectRepository extends EntityRepository
+class RedirectRepository extends EntityRepository implements RedirectRepositoryInterface
 {
     /**
-     * @param string $source
-     *
-     * @return RedirectInterface|null
-     *
-     * @throws \Doctrine\ORM\NonUniqueResultException
+     * {@inheritdoc}
      */
     public function findBySource(string $source): ?RedirectInterface
```

Comment:
We probably need to remove this method since it shouldn't be used

Reply:
Yeah definitely